src/OpenTelemetry.Instrumentation.AspNet/README.md

```diff
@@ -38,24 +38,23 @@ Configuration with ASP.NET (Full .NET Framework) running in IIS or IIS Express
     ```csharp
     public class WebApiApplication : HttpApplication
     {
-        private TracerFactory tracerFactory;
+        private TracerFactory tracerProvider;
```

Comment:
This must be `TracerProvider`. Or can just use IDisposable as well.

Reply:
```suggestion
        private TracerProvider tracerProvider;
```